libs/checkpoint-sqlite/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "langgraph-checkpoint-sqlite"
+version = "1.0.0"
+description = "Library with a SQLite implementation of LangGraph checkpoint saver."
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://www.github.com/langchain-ai/langgraph"
+packages = [{ include = "langgraph" }]
+
+[tool.poetry.dependencies]
+python = "^3.9.0,<4.0"
+aiosqlite = "^0.20.0"
```

Comment:
lets update

Reply:
i think it's the latest https://pypi.org/project/aiosqlite/

Reply:
ah cool